### DISCUSSION TITLE
大佬请问当日更新里面的清单是带密钥的吗？

### DISCUSSION THREAD

**Discussion by beefdinner12** [pjy612/SteamManifestCache#581] [discussion]
我看清单列表里面有太子沉香，但是我在cmd里面输入appid之后显示密钥缺失-跳过解锁，然后我用greenluma/steamtools解锁之后还是无法下载，这种情况是什么原因导致的呢，是需要内测版才可以从steam里面开启下载吗？提前说一声抱歉，wiki和介绍都看了很多遍了但还是不太明白

**Comment by pjy612** [comment]
当日更新只是收录游戏的更新情况，只针对清单
密钥自己想办法 ，比如去其他仓库提取或者 解锁插件里面提取。
